install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -814,10 +862,10 @@
         "logo": "{{.IdeLogoBase}}/clionLogo.svg",
         "image": "{{.Repository}}/ide/clion:{{.WorkspaceVersions.Workspace.DesktopIdeImages.CLionImage.Version}}",
         "latestImage": "{{.ResolvedJBImageLatest.CLion}}",
-        "pluginImage": "{{.JetBrainsPluginImage}}",
+        "pluginImage": "{{.JetBrainsPluginImagePrevious}}",
```

Comment:
`CLion` and `RustRover` 2024.3 are not release yet, we need to force them to use old backend plugin